Add guidance for postcode format filter and 8.2.0 update notes (#465)
nhsuk/nhsuk.service-manual.prototype-kit.docs: app/guides/filters.md

{% raw %}<p>Your NHS number is 999 123 4567.</p>{% endraw %}
```

### formatPostcode

> Use this to format an UK postcode so that it appears uppercase with a space in the middle, regardless of how the user entered it.

Reviewer: ```suggestion
Use this to format a UK postcode so that it appears uppercase with a space in the middle, regardless of how the user entered it.
```

> Any UK postcodes will be formatted, otherwise the input will be unchanged.

Reviewer: ```suggestion
Only UK postcodes will be formatted, otherwise the input will stay unchanged.
```

> This does not validate that the UK postcode is a real one in use.

Reviewer: I think "validate… is a real one" was unclear and I don't know what "in use" means?

```suggestion
This filter does not check for valid UK postcodes.
```

Author: I was trying to avoid "valid" as that can be ambiguous - could mean either a valid format, or valid in the sense that that specific postcode has been assigned to an address in use in the real world.

How about

> This filter does not check that the UK postcode is actually assigned to a real address.

Reviewer: I suppose it's the opposite, is an address is assigned to a postcode?

> This filter does not check that the UK postcode actually exists.

Author: I like that. Have updated.

> This does not validate that the UK postcode is a real one in use.

Author: ```suggestion
This filter does not check that the UK postcode actually exists.
```
